tracker/src/main/java/org/matomo/sdk/dispatcher/DefaultDispatcher.java

```diff
@@ -204,7 +272,7 @@ public void run() {
     };
 
     private boolean isConnected() {
```

Comment:
If we go for `DispatchMode.OFFLINE` refactoring this to `isOnline()` might be a bit clearer, maybe `assumeOnline()`, thoughts?

Reply:
I agree, calling the method `isOnline` would make more sense.  I'm not sure I understand `assumeOnline` though.

Reply:
Let's just rename this `isOnline`.

Reply:
Will do.